OpenRA.Platforms.Default/Shader.cs

```diff
@@ -24,6 +24,7 @@ class Shader : ThreadAffine, IShader
 
 		readonly Dictionary<string, int> samplers = new Dictionary<string, int>();
 		readonly Dictionary<int, ITexture> textures = new Dictionary<int, ITexture>();
+		readonly List<int> unbindTextures = new List<int>();
```

Comment:
Could save 1 line by having this as Queue! :P

Reply:
Not sure what you mean. Can you elaborate?

Reply:
https://github.com/teinarss/OpenRA/commit/054386f411340c91fb13a298e8ec6daad1378359